src/modules/user/index.ts

```diff
@@ -358,8 +358,54 @@ const getAvailableTimeZones = async (): Promise <any>  => {
   }
 }
 
+async function getEComStores(token: any, baseURL: string, vSize = 100, facilityId?: string): Promise<Response> {
+  const filters = {} as any;
+
+  if(facilityId) {
```

Comment:
If facilityId is not found, then we should not fetch productStores, as here we are fetching data from ProductStoreFacilityDetail, so if we do not have facilityId then there is no use case of getting data from this entity.